doc(release) Release notes map 23.10.23 mbi 23.10.15 (#4684)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-commercial-extensions.mdx

## Centreon MAP

### 23.10.23

Release date: `September 29, 2025`

<details>
  <summary>Bug fixes</summary>

> - [Config] The configure.sh script does not start the mysql service anymore.
> - [Editor] Fixer an issue that caused links to be positioned differently in the editor when migrating from MAP Legacy to MAP.
> - [Editor] Fixed issues when copy/pasting elements.

Reviewer: ```suggestion
- [Config] Fixed an issue where the configure.sh script was not starting the mysql service anymore.
- [Editor] Fixed an issue that caused links to be positioned differently in the editor when migrating from MAP Legacy to MAP.	
- [Editor] Fixed issues occurring when copying/pasting elements.
```

> - [Viewer] Fixed issue that prevented pie chart widget from being displayed in percentage per status.

Reviewer: ```suggestion
- [Viewer] Fixed an issue that prevented pie chart widgets from being displayed in percentage per status.
```

## Centreon MBI

### 23.10.15

Release date: `September 29, 2025`

<details>
  <summary>Bug fixes</summary>

> - [ETL] Fixed issue when metric names exceed 255 characters.
> - [MBI] Fixed an issue causing ACLs not to be applied on Report designs.	

Reviewer: ```suggestion
- [ETL] Fixed an issue occurring when metric names exceeded 255 characters.
- [MBI] Fixed an issue causing ACLs not to be applied to report designs.	
```
